requirements.txt

```diff
@@ -3,7 +3,7 @@ flask-restplus==0.12.1
 marshmallow==3.0.0rc6
 jinja2==2.11.3
 Flask==1.0.2
-pyyaml==5.4
```

Comment:
Otherwise we encounter the following error while installing pip dependencies.

![image](https://github.com/idealista/prom2teams/assets/1327782/ef2ca01f-fdd0-4d95-90ec-9c261cb9b5d7)